scripts/dependencies/tools.ts

```diff
@@ -147,10 +147,15 @@ export class KuberlrAndKubectl implements Dependency {
     await fs.promises.symlink('kuberlr', binKubectlPath);
   }
 
-  async getAvailableVersions(): Promise<string[]> {
+  async getAvailableVersions(includePrerelease = false): Promise<string[]> {
     const tagNames = await getPublishedReleaseTagNames(this.githubOwner, this.githubRepo);
+    const allVersions = tagNames.map((tagName: string) => tagName.replace(/^v/, ''));
+
+    if (!includePrerelease) {
+      return allVersions.filter(version => semver.prerelease(version) === null);
```

Comment:
Note that `semver.prerelease('pikachu')` (i.e. invalid version) is `null`; that might be okay.

Reply:
I can see why this stood out to you - we don't validate the versions on most `Dependency`s at the moment. This wouldn't be bad to have. Of course, the call to `semver.prerelease` isn't the place to do version validation.

I want to leave this as it is for the moment. The `Dependency`s with weird versions (such as Wix) have version validation. And I don't think it's likely that the others will be published with invalid versions. I can add it if you want me to though.